package.json

```diff
@@ -91,7 +91,7 @@
     "build": "npm run generateParams && npm run clean && tsc && cp src/*.html package*.json README.md ./dist",
     "clean": "rm -rf dist",
     "generateParams": "node --loader ts-node/esm ./generateOptionsFromSpec.ts",
-    "start": "node src/index.js",
+    "start": "node dist/index.js",
```

Comment:
Wasn't sure about this. Happy to revert, but I _think_ this is what we want for testing it locally, e.g `npm run build && npm start -- projects list`

Reply:
Good catch! I wasn't really using this command, so never really noticed it was outdated!